cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,78 @@
+import PageObject from '../PageObject';
+
+class SettingsPageObject extends PageObject {
+    url = '/#/settings';
+
+    get usernameField() {
+        return cy.getByDataCy('username-settings-field')
+    }
+
+    get bioField() {
+        return cy.getByDataCy('bio-settings-field')
+    }
+
+    get updateSettingsBtn() {
+        return cy.getByDataCy('update-settings-btn')
+    }
+
+    get emailField() {
+        return cy.getByDataCy('email-settings-field')
+    }
+
+    get passwordField() {
+        return cy.getByDataCy('password-settings-field')
+    }
+
+    get LogOutBtn() {
+        return cy.getByDataCy('logout-settings-btn')
+    }
+
+    typeUsername(updateUsername) {
+        this.usernameField
+            .type(updateUsername);
+    }
+
+    typeBio(updateBio) {
+        this.bioFieldField
```

Comment:
There is a typo in the `typeBio` method. It references `this.bioFieldField` instead of `this.bioField`. Correct this to ensure the method functions properly.